index.html

```diff
@@ -58,9 +69,6 @@
         </div>
```

Comment:
I believe this is an extra div closing tag. I don't see an associated opening div tag.